src/main/kotlin/sg/com/quantai/etl/controllers/BbcNewsTransformationController.kt

```diff
@@ -0,0 +1,23 @@
+package sg.com.quantai.etl.controllers
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.*
+import sg.com.quantai.etl.services.BbcNewsTransformationService
+import org.springframework.http.ResponseEntity
+import org.springframework.http.HttpStatus
+
+@RestController
+@RequestMapping("/transform_bbc")
+class BbcNewsTransformationController(
```

Comment:
Lines 9 and 10

The scope of the data is still the same: BBC News.
Therefore, a new controller is not necessary.